### PR TITLE
Add genre field to artist table and added script to create and initialise database 

### PR DESCRIPTION
**Pull request by toiletspider**
### Description:
Artist Database: Add genre to artists table in database.

Created shell script to create and initialise database. This should decrease the fragmented setup for the database creation.

### Migrations
* 20190905000000_add_genre_field_to_artist

### Environment Variables
 * No change

### CODE REVIEW

File: db/migrations/20190905000000_add_genre_field_to_artist/up.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+alter table artists
```
Comment (mikethetike) [inline db/migrations/20190905000000_add_genre_field_to_artist/up.sql]: Please add to the summary of the PR

File: db/src/models/artists.rs [hunk 2]
```diff
@@ -35,6 +35,7 @@ pub struct Artist {
     pub created_at: NaiveDateTime,
     pub updated_at: NaiveDateTime,
     pub other_image_urls: Option<Vec<String>>,
+    pub genres_id: Option<Uuid>,
```
Comment (mikethetike) [inline db/src/models/artists.rs]:
```suggestion
    pub main_genre_id: Option<Uuid>,
```

File: db/migrations/20190905000000_add_genre_field_to_artist/up.sql [hunk 3]
```diff
@@ -0,0 +1,2 @@
...
+    add genres_id UUID NULL REFERENCES genres(id);
```
Comment (mikethetike) [inline db/migrations/20190905000000_add_genre_field_to_artist/up.sql]:
```suggestion
    add main_genre_id UUID NULL REFERENCES genres(id);
```

File: db/migrations/20190905000000_add_genre_field_to_artist/up.sql (hunk 1 above)
Comment (toiletspider) [reply db/migrations/20190905000000_add_genre_field_to_artist/up.sql]: Added migrations names

Comment (Timo614) [inline db/migrations/20190905000000_add_genre_field_to_artist/up.sql]: We should add a down migration for this change as well.

Comment (toiletspider) [reply db/migrations/20190905000000_add_genre_field_to_artist/up.sql]: Added migration name to the RP description

File: db/migrations/20190905000000_add_genre_field_to_artist/up.sql (hunk 3 above)
Comment (toiletspider) [reply db/migrations/20190905000000_add_genre_field_to_artist/up.sql]: Renamed table colom

File: db/src/models/artists.rs (hunk 2 above)
Comment (toiletspider) [reply db/src/models/artists.rs]: Renamed table colom

**Comment by Krakaw** [comment]
@toiletspider please also add the issue link that this PR references.

**Pull request merged**